### ISSUE TITLE
The model seems to be different from that in the paper.

### ISSUE THREAD

**Issue by lukun199** [csjcai/SICE#10]
Thank you for your work, and I'm going to cite this. (I used the dataset actually.)
But when I want to calculate the parameters in the model, I check the prototxt file as the README  file writes:

> Network structure: *.prototxt (to view the network structure, use [this link](http://ethereon.github.io/netscope/#/editor))

However, this structure is quite different from that in the paper (e.g., the kernel size, stride, number of convolutional layers. See below). 

By the way, since the model is originally implemented with tensorflow, is there any plan to release the tensorflow version? The caffe model requires so much memory(~64G).

>gradient descent (SGD) with a batch size of 80 patches is used in training. We implement our model using the TensorFlow package. The momentum parameter and weight decay parameter are set to 0.9 and 0.0001, respectively. 

![image](https://user-images.githubusercontent.com/48766152/68988667-ebe3eb80-0875-11ea-85b6-4799af1bacf2.png)
